GUI/Main.cs

```diff
@@ -979,11 +978,11 @@ private void exportModListToolStripMenuItem_Click(object sender, EventArgs e)
 
         private void selectKSPInstallMenuItem_Click(object sender, EventArgs e)
         {
-            Main.Instance.Manager.ClearAutoStart();
-            var old_instance = Main.Instance.CurrentInstance;
+            Instance.Manager.ClearAutoStart();
+            var old_instance = Instance.CurrentInstance;
             var result = new ChooseKSPInstance().ShowDialog();
-            if (!Equals(old_instance, Main.Instance.CurrentInstance))
-                Main.Instance.CurrentInstanceUpdated();
+            if (result == DialogResult.OK && !Equals(old_instance, Instance.CurrentInstance))
```

Comment:
Was there a bug here? We weren't checking to see if the user clicked OK or Cancel?